src/pycountry/__init__.py

```diff
@@ -72,20 +75,18 @@ def add_result(country: "pycountry.db.Country", points: int) -> None:
             pass
 
         # Prio 2: exact matches on subdivision names
-        match_subdivions = pycountry.Subdivisions.match(
-            self=subdivisions, query=query
-        )
-        for candidate in match_subdivions:
-            print(candidate)
-            add_result(candidate.country, 49)
+        match_subdivions = subdivisions.match(query=query)
+        for subdivision in match_subdivions:
```

Comment:
mypy didn't like `candidate` being re-used for different variables types so I renamed these to use separate names per type.